test/controllers/api/jobs_controller_test.rb

```diff
@@ -30,6 +30,8 @@ class Api::JobsControllerTest < ActionController::TestCase
       priority: 1,
       retry_max: 10,
       retry_delay: 300,
+      original: 'file://this/aint/real.fake',
+      original_format: 'mp3',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.